views/js/previewer/plugins/tools/scale/component/scss/devicesPreviewer.scss

```
@@ -61,3 +61,12 @@
         }
     }
 }
+
+.item-writing-mode-vertical-rl {
+    & .preview-container,
+    & .preview-frame,
+    & .preview-content,
+    & #qti-content {
+         height: 100%;
+     }
```

```suggestion
    .preview-container,
    .preview-frame,
    .preview-content,
    #qti-content {
        height: 100%;
    }
```
Looks like the convention is to use `&` only when referencing the same element or specific relationships.